api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/InventoryServiceClient.java

```diff
@@ -273,15 +264,6 @@ public Mono<Void> deleteInventoryByInventoryId(String inventoryId){
     }
 
 
-    public Mono<InventoryResponseDTO> addProductToInventoryByName(String inventoryName, ProductRequestDTO productRequestDTO) {
-        return webClient.post()
-                .uri(inventoryServiceUrl + "/{inventoryName}/products/by-name", inventoryName)
-                .body(Mono.just(productRequestDTO), ProductRequestDTO.class)
-                .retrieve()
-                .bodyToMono(InventoryResponseDTO.class);
-    }
-
-
```

Comment:
I see you removed the functionality to remove products inside an inventory through its inventory name, which is good because using the name string is risky in the situation where 2 inventories are created with the same name.